src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -2164,4 +2164,10 @@ From there, simply click on one of the supported files in the File Explorer and
   <data name="AlwaysOnTop_RoundCorners.Content" xml:space="preserve">
     <value>Enable round corners</value>
   </data>
-</root>
+  <data name="FileExplorerPreview_ToggleSwitch_Monaco_Try_Format.Description" xml:space="preserve">
+    <value>Applies to json and xml</value>
+  </data>
+  <data name="FileExplorerPreview_ToggleSwitch_Monaco_Try_Format.Header" xml:space="preserve">
+    <value>Try to format the source</value>
```

Comment:
```suggestion
    <value>Try to format the source for presentation</value>
```

Reply:
I personally prefer format over beautify here. If we decide to go for beaufity I think the code should be aligned too.
Also VS Code use format/formatting.

Reply:
Yeah, "beautify" sounds like we're putting make-up on it xD

Reply:
The let's keep `formatting`, but add `for presentation`.

Reply:
For preview? Sounds better?